src/Auditor.jl

```diff
@@ -166,6 +166,7 @@ function audit(prefix::Prefix; io=stderr,
             if !silent
                 warn(io, "Skipping abspath scanning of $(f), as we can't open it")
             end
+        catch
```

Comment:
@staticfloat was that `except` above supposed to be a `catch`?

Reply:
sigh.  yes.  snake-brain and insufficient test coverage, apparently.  I'll add a test case for this.

Reply:
How do I ask femtocleaner to rebase on latest master?

Reply:
There's currently no way to do that. I should add one. In the meantime, the easiest is probably to close this and open a new `Run femtocleaner issue`.